extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -162,9 +162,22 @@ export class DoubleclickA4aEligibility {
    */
   isA4aEnabled(win, element, useRemoteHtml) {
     this.unconditionedExperimentSelection(win, element);
-    if ((useRemoteHtml && !element.getAttribute('rtc-config')) ||
-        'useSameDomainRenderingUntilDeprecated' in element.dataset ||
-        element.hasAttribute('useSameDomainRenderingUntilDeprecated')) {
+    const hasUSDRD =
+          'useSameDomainRenderingUntilDeprecated' in element.dataset ||
+          element.hasAttribute('useSameDomainRenderingUntilDeprecated');
+    if (hasUSDRD) {
+      user().warn(TAG, 'useSameDomainRenderingUntilDeprecated will no longer ' +
```

Comment:
Recommend making a generic string you macro into to save duplicate strings:
const usdr = 'useSameDomainRenderUntilDeprecated';
const hasUSDRD = usdr in element.dataset || element.hasAttribute(usdr);
const warnDeprecation = feature => user.warn(TAG, `The user of ${feature} will no longer...`});
if (hasUSDRD) warnDeprecation(usdr);

Reply:
done